### ISSUE TITLE
Add Public Methods Category to Controls

### ISSUE THREAD

**Issue by break-stuff** [break-stuff/wc-storybook-helpers#26]
Even though methods are not "controllable", adding a "Methods" section to the controls would be very beneficial for documentation purposes.

**Comment by rstoneIDBS** [comment]
I knew I had seen this mentioned somewhere ;) Was just looking through the docs trying to figure out how to enable this. It would be extra handy if it included both instance and static methods.

**Comment by rstoneIDBS** [comment]
I did have a look to see if this was something I could contribute to, but it quickly gets very complicated when you start considering documenting the method parameters + inherited methods + private/public methods so would require a fair amount of understanding about the CEM format and how to wrangle what is needed out of Storybook.

tldr; it scared me off ;)

**Comment by break-stuff** [comment]
This has been added to the new version of this package:
https://wc-toolkit.com/integrations/storybook/
